src/Microsoft.AspNetCore.Blazor.Build.Analyzers/Microsoft.AspNetCore.Blazor.Build.Analyzers.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Do we want to call this `Microsoft.AspNetCore.Blazor.Analyzers` instead? Do you think we'd ever have more than one package of analyzers?

Reply:
Good point. I'll rename.